Lib/test/test_tools/test_i18n/test_i18n.py

```diff
@@ -1,6 +1,8 @@
 """Tests to cover the Tools/i18n package"""
 
 import os
+from pathlib import Path
```

Comment:
Is it necessary to use `pathlib`? Other tests simply use `os.path`.

Reply:
There were only a handful of uses of `os` so I went ahead and replaced them with `pathlib` which I think improves readability, but I'm happy to revert the change if you prefer to keep `os` :)